modules/url-check.js

```diff
@@ -1,14 +1,11 @@
 const { red } = require('kleur')
-
+const regex = /http[s]?:\/\//g
 module.exports = () => {
   if (!url) {
     console.log(red('No website was provided.'))
     process.exit(1)
   }
-  if (url.startsWith('http://')) {
-    url = url.slice(7)
+  if (url.match(regex)) {
+    url = url.replace(url.match(regex)[0],'')
```

Comment:
This can be simplified, see https://codepen.io/DanielRuf/pen/eYYRNYz?editors=1111